src/components/tx/TxData.js

```diff
@@ -121,6 +129,50 @@ class TxData extends React.Component {
     this.setState({ children: !this.state.children });
   }
 
+  /**
+   * Given a graph type, it calculates the DAG (verification or funds) and renders into the HTML element
+   *
+   * @param {string} graphType
+   */
+  calculateNeighborsGraph = async (graphType) => {
+    const viz = new Viz({ Module, render });
+
+    const graphvizResponse = await graphvizApi.dotNeighbors(this.props.transaction.hash, graphType);
+    const element = await viz.renderSVGElement(graphvizResponse);
+
+    element.id = `graph-${graphType}`;
```

Comment:
I know we won't recalculate this, but we shouldn't add an element with the same id as the parent. Maybe use `graph-${graphType}-data`.